sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/models/student-scholastic-standings.dto.ts

```diff
@@ -35,3 +35,30 @@ export class ScholasticStandingSubmittedDetailsAPIOutDTO extends IntersectionTyp
   ScholasticStandingData,
   ActiveApplicationDataAPIOutDTO,
 ) {}
+
+/**
+ * Represents the possible errors that can happen during the
+ * application bulk withdrawal and provides a detailed description
+ * for every record that has an error.
+ */
+export interface ApplicationBulkWithdrawalValidationResultAPIOutDTO {
+  recordIndex: number;
+  sin?: string;
+  applicationNumber?: string;
+  withdrawalDate?: Date;
+  errors: string[];
+  infos: ValidationResultAPIOutDTO[];
+  warnings: ValidationResultAPIOutDTO[];
+}
+
+/**
+ * Represents an error considered not critical for
+ * an offering or application withdrawal
+ * and provides an user-friendly message
+ * and a type that uniquely identifies this warning
+ * or info.
+ */
+class ValidationResultAPIOutDTO {
+  typeCode: string;
```

Comment:
type code could be an enum.

Reply:
I am not using the info and warning, that I use the ValidationResultAPIOutDTO. In the future, when there are business validations, this can be enhanced.